controllers.js

```diff
@@ -35,5 +35,19 @@ function getArticles(req, res, next) {
     .catch((err) => next(err))
 }
 
+function getCommentsByArticleId(req, res, next) {
+    const articleId = req.params.article_id
+    const promises = [ selectArticleById(articleId), selectCommentsByArticleId(articleId)]
+
+    Promise.all(promises) 
+    .then((results) => {
+        if (results[1].length === 0) {
+            res.status(201).send({ comments: results[1]});
```

Comment:
This should be a 200 - nothing has been created here

Reply:
Also, if you get to this point, you can just send back the comments anyway? Both conditions here are sending back the same thing.